scripts/deployment/phase2-assets/collaterals/deploy_aave_v3_usdc.ts

```diff
@@ -49,7 +45,10 @@ async function main() {
   /********  Deploy Aave V3 USDC wrapper  **************************/
 
   const StaticATokenFactory = await hre.ethers.getContractFactory('StaticATokenV3LM')
```

Comment:
lets add a check in this file that we're targetting mainnet (just like we have a check in the ausdbc file that we're targetting base

Reply:
But cant USDC be deployed at some point to Base?